content/get-started/get-started.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Getting Started
+seo-title: Getting Started
```

Comment:
I think we used the past tense `get started` everywhere, wondering if we should be consistent?

Reply:
It looks like we mix the tenses. We should choose one or the other, what do you recommend?

 - Running a node / Run a node
 - Getting started / Get started
 - Setting up an account / Set up an account
 - Transacting / Transactions

Reply:
I like running more than run. 

Instead of transactions I like Sending Transactions

Reply:
I actually decided to use Get Started because all of our navigation is already in that tense. It's better to be consistent and negotiate a larger change later.